test/e2e/kamsuite/kamsuite.go

```diff
@@ -139,6 +144,24 @@ func deleteGitlabRepository(arg []string) (bool, string) {
 	return true, stderr.String()
 }
 
+func ClearDir() error {
```

Comment:
```suggestion
func clearDir() error {
```
The function is exported but only used within the same package